src/styles/blocks/card.css

```
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+
+  padding: 32px 16px 16px;
+
+  background: #fff;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+
+  margin: 0 3px 40px;
```

better use auto for vertical paddings for auto-centering
```suggestion
  margin: 0 auto 40px;
```